docs/guides/use-connectors-in-hybrid-mode.md

```diff
@@ -0,0 +1,103 @@
+
+Run the following script:
+
+```shell
+docker run --rm --name=HybridConnectorRuntime \
+    -e ZEEBE_CLIENT_SECURITY_PLAINTEXT=false \
+    -e ZEEBE_CLIENT_CLOUD_CLUSTER-ID='<YOUR_CLUSTER_ID>' \
+    -e ZEEBE_CLIENT_CLOUD_CLIENT-ID='<YOUR_CLIENT_ID>' \
+    -e ZEEBE_CLIENT_CLOUD_CLIENT-SECRET='<YOUR_CLIENT_SECRET>' \
+    -e ZEEBE_CLIENT_CLOUD_REGION='<YOUR_CLUSTER_REGION>' \
+    -e CAMUNDA_OPERATE_CLIENT_URL='https://<region>.operate.camunda.io/<cluster-id>' \
+    -e CONNECTOR_HTTP_REST_TYPE='io.camunda:http-json:local' \
+        camunda/connectors-bundle:<desired-version>
+```
+
+### Option B: build your own runtime
+
+Use this option if you need to make code modifications.
+
+- Make sure `docker` is installed.
+- Clone https://github.com/camunda/connectors.
+- Go to `<connectors-root>/bundle/default-bundle`.
+- Build Connectors image, e.g. `docker build -f Dockerfile -t camunda/connectors-bundle:<desired-version> .`.
+- Run the same `docker run ...` command as in [Option A](#option-a-get-connectors-runtime-from-docker-registry).
+
+### Explanation
+
+Pay attention at the line `-e CONNECTOR_HTTP_REST_TYPE='io.camunda:http-json:local'`. This line means that we have to override
+`CONNECTOR_X_TYPE` with given type. In this case, we want to register local self-managed HTTP REST connector as `io.camunda:http-json:local`.
+The `X` is normalized to environment variable Connector name. For example, the [HTTP REST Connector](https://github.com/camunda/connectors/blob/main/connectors/http/rest/src/main/java/io/camunda/connector/http/rest/HttpJsonFunction.java#L33)
+`HTTP REST` name becomes `HTTP_REST`, or the [Kafka Consumer Connector](https://github.com/camunda/connectors/blob/main/connectors/kafka/src/main/java/io/camunda/connector/kafka/inbound/KafkaExecutable.java#L20) name
+transformed to `KAFKA_CONSUMER`, therefore to override it one would need to pass `CONNECTOR_KAFKA_CONSUMER_TYPE=xxx` environment variable.
+
+## Modify element template
+
+As mentioned before, to relate Connector element templates with Connectors runtime, you have to modify the task definition type.
+
+In order to do that, get a copy of the element template you wish to override. All latest versions of the official element
+templates can be found at the [official Connectors repository](https://github.com/camunda/connectors) at path `connectors/<desired connector>/element-templates/`.
+Then, do the following:
+
+- [Create new or import](../../components/connectors/manage-connector-templates/) desired Connector element template into your Web Modeler.
+- Modify the `value` to desired new type of the property `zeebe:taskDefinition:type` for outbound Connectors, or `inbound.type` for inbound ones.
+- Publish new element template.
+- Use new published template in the BPMN diagram as usual.
+
+For example, if you defined `CONNECTOR_HTTP_REST_TYPE='io.camunda:http-json:local'` in runtime, you have to do the following in the
```

Comment:
should we describe here the options the have:
making the id visible
Adding a dropdown for the type (run in SM and run in SAAS)

At the end it is totally up to the user, but i think it would be valuable to show here the options.
- duplicating template and hiding the id
- Adding dropdown to a REST
- make it visible and collapse the group

Reply:
If you think, it's valuable, we can add it. For now, we pre-created hybrid examples (see appendix section), but can also describe those options too.

Reply:
Done